### ISSUE TITLE
Starting Worker cause crash with AIR 33.1

### ISSUE THREAD

**Issue by itlancer** [Gamua/Adobe-Runtime-Support#249]
## Problem Description
Starting `Worker` cause crash with AIR 33.1.
Tested with AIR 33.1.0.50 with multiple Windows, Android, macOS and iOS devices.
Cannot test it with tvOS because of this issue: https://github.com/Gamua/Adobe-Runtime-Support/issues/234
HARMAN AIR license file used for builds.
Same problem in all cases.

It works fine with AIR 33.0.2.338 and below.

Related issues (not the same):
https://github.com/Gamua/Adobe-Runtime-Support/issues/240
https://github.com/Gamua/Adobe-Runtime-Support/issues/231
https://github.com/Gamua/Adobe-Runtime-Support/issues/96

## Steps to Reproduce
Launch code below with Windows, Android, macOS or iOS devices.

Application example with sources attached.
[worker_start_crash.zip](https://github.com/Gamua/Adobe-Runtime-Support/files/4274995/worker_start_crash.zip)

```
package {
	import flash.display.Sprite;
	import flash.events.Event;
	import flash.system.Worker;
	import flash.system.WorkerDomain;
	
	public class WorkerStartCrash extends Sprite {
		
		public function WorkerStartCrash() {
			if (Worker.current.isPrimordial) {
				addEventListener(Event.ADDED_TO_STAGE, init);
			} else {
				trace("worker");
			}
		}
		
		private function init(e:Event):void {
			removeEventListener(Event.ADDED_TO_STAGE, init);
			
			var worker:Worker = WorkerDomain.current.createWorker(this.loaderInfo.bytes);
			worker.start();//This line cause crash
		}
	}
}
```

**Actual Result:**
Application crash.
With Windows:
```
Problem signature:
  Problem Event Name:	APPCRASH
  Application Name:	adl.exe
  Application Version:	33.1.0.50
  Application Timestamp:	5e4e2cf2
  Fault Module Name:	Adobe AIR.dll
  Fault Module Version:	33.1.0.50
  Fault Module Timestamp:	5e4e3637
  Exception Code:	c0000005
  Exception Offset:	004495eb
  OS Version:	6.3.9600.2.0.0.256.48
  Locale ID:	1049
  Additional Information 1:	5861
  Additional Information 2:	5861822e1919d7c014bbb064c64908b2
  Additional Information 3:	d1d9
  Additional Information 4:	d1d94a13d3609d6b740644c12508f581
```

With Android:
```
W/InputDispatcher: channel '16039a6 worker.start.crash/worker.start.crash.AppEntry (server)' ~ Consumer closed input channel or an error occurred.  events=0x9
E/InputDispatcher: channel '16039a6 worker.start.crash/worker.start.crash.AppEntry (server)' ~ Channel is unrecoverably broken and will be disposed!
I/android_os_HwBinder: HwBinder: Starting thread pool for default::android.hardware.wifi.supplicant@1.0::ISupplicant
I/WindowManager: WIN DEATH: Window{16039a6 u0 worker.start.crash/worker.start.crash.AppEntry}
W/InputDispatcher: Attempted to unregister already unregistered input channel '16039a6 worker.start.crash/worker.start.crash.AppEntry (server)'
I/ActivityManager: Process worker.start.crash (pid 6875) has died: vis  +99TOP 
W/libprocessgroup: kill(-6875, 9) failed: No such process
Zygote: Process 6875 exited due to signal (11)
```

With macOS just "silent" crash without any errors.

**Expected Result:**
Application correctly start `Worker` without crashes.

## Known Workarounds
none

**Comment by ajwfrost** [comment]
Thanks - will get a fix for this one out as soon as we can..

**Comment by itlancer** [comment]
Fixed with AIR 33.1.1.63.
Thank you!
